week-02-ajax-and-js-objects/day-02/drill/readme.md

```diff
@@ -0,0 +1,48 @@
+## HTML Strings
+
+For the following challanges you are going to use the template from https://github.com/sf-wdi-24/frontend-template
+
+1. From the template repo copy the url to clone it
+2. In your terminal, go into your dev folder and type 'git clone' + paste the link
+3. Open this up in Sublime
+4. In `main.js` change the file from:
+    ```
+    // wait for DOM to load before running JS
+    $(function() {
+
+    // check to make sure JS is loaded
+    console.log('JS is loaded!');
+
+    // your code here
+
+    });
+   ```
+   to:
+   ```
+   console.log("Sanity Check: JS is working!");
+
+   $(document).ready(function(){
+
+     // code in here
+
+     });
+     ```
+4. Paste the data object into the JS file.
+5. Type your solution into the document.ready method.
```

Comment:
I think the order/formatting here is a little confusing b/c you're saying to type the solution before stating the goal. I think a clear title of "challenges" on line 36 would help as well as some numbering for the challenges we want them to complete.